selfdrive/modeld/dmonitoringmodeld.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
```

Comment:
can you make this executable?

Reply:
I think there's a pre-commit hook to check for this. @jnewb1 want to look into setting that up?